src/main/java/Game/Game.java

```diff
@@ -86,6 +89,13 @@ public void paintComponent( Graphics g )
 			lastTimeAlive = timeAlive;
 		}
 		
+		// updates health label and changes color if low health
+		lblHealth.setText("Health: " + m_ship.getShipHealth());
+		if(m_ship.getShipHealth() < 10) {
```

Comment:
Since you initialize health to be 10 anyways, the ship health will always be red. Did you mean to initialize to 10?